### PR TITLE
Handling CORS

### PR DESCRIPTION
**Pull request by nirgn975**
#39

### CODE REVIEW

**Comment by nirgn975** [comment]
@ordavidil Ready for code review.

**Comment by nirgn975** [comment]
@ordavidil Ready for code review.

File: server/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+psycopg2==2.6.1
+Pillow==2.9.0
+gunicorn==19.3.0
 django-extensions==1.6.6
+
+# Rest Framework
 djangorestframework==3.3.3
-psycopg2==2.6.1
+Markdown==2.6.6
+django-filter==0.13.0
```
Comment (ordavidil) [inline server/requirements.txt]: @nirgn975 
where is the usage of this app ?

File: server/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
 Django==1.9.5
+psycopg2==2.6.1
+Pillow==2.9.0
+gunicorn==19.3.0
 django-extensions==1.6.6
+
+# Rest Framework
 djangorestframework==3.3.3
-psycopg2==2.6.1
+Markdown==2.6.6
```
Comment (ordavidil) [inline server/requirements.txt]: @nirgn975 
where is the usage of this app ?

Comment (nirgn975) [inline server/requirements.txt]: ```pip install markdown       # Markdown support for the browsable API.``` - [http://www.django-rest-framework.org/#installation](http://www.django-rest-framework.org/#installation)

File: server/requirements.txt (hunk 1 above)
Comment (nirgn975) [inline server/requirements.txt]: ```pip install django-filter  # Filtering support``` - [http://www.django-rest-framework.org/#installation](http://www.django-rest-framework.org/#installation)

**Comment by nirgn975** [comment]
I Checked and that is a problem in the response not in the request, so that's a django problem and not a angular one. This is the easy why to handle CORS, I think we should use it.

**Pull request merged**